### ISSUE TITLE
Latest SWB Fails on Firefox 44.02

### ISSUE THREAD

**Issue by Stoft1** [publiclab/spectral-workbench#326]
1) Video w/spectrum colors do appear in small window but on clicking 'Capture' no signals appear and display of data is either left-shifted or cut off at 520nm. Seems an old Cal is being used (improperly) for this display. This bug was previously reported -- but still exists in this new relase of SWB.

2) In attempting to 'reset' this issue, the 'New Calibration' option was selected and 'Switch cal' was triggered -- only to get an error page with a link to report the issue.

So, in short, SWB is DOA on Firefox.

**Comment by jywarren** [comment]
Hi, Dave - the issue is still open here: https://github.com/publiclab/spectral-workbench/issues/285 -- I'm going to close this new issue and consolidate your feedback there.
